src/utils/formatTokenAmount.ts

```diff
@@ -10,11 +10,11 @@ export function formatTokenAmount(amount: CurrencyAmount<Currency> | undefined,
     return '0'
   }
 
-  if (parseFloat(amount.toFixed(Math.min(sigFigs, amount.currency.decimals))) < 0.0001) {
-    return '<0.0001'
+  if (parseFloat(amount.toSignificant(Math.min(sigFigs, amount.currency.decimals))) < 0.000001) {
```

Comment:
probably cleaner to do something like `amount.divide(amount.decimalScale).lessThan(new Fraction(1, 100000))`